src/runtime_src/core/common/query_requests.h

```diff
@@ -96,9 +96,18 @@ enum class key_type
   expected_sc_version,
   xmc_status,
   xmc_reg_base,
+  xmc_scaling_support,
   xmc_scaling_enabled,
   xmc_scaling_override,
   xmc_scaling_temp_override,
+  xmc_scaling_critical_pow_threshold,
+  xmc_scaling_critical_temp_threshold,
+  xmc_scaling_threshold_power_limit,
+  xmc_scaling_threshold_temp_limit,
+  xmc_scaling_threshold_power_override_en,
+  xmc_scaling_threshold_temp_override_en,
```

Comment:
what does "en" stand for?

Reply:
enable

Reply:
So this value would be true or false?
I understand that the name is already very long, but it is ok to be verbose here. Please change en to enable